.deploy/nais.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
 
 spec:
   image: {{ image }}
+  leaderElection: true
```

Comment:
Vet vi har slitt litt med leaderElection tidligere (ref problemer i mottak for lenge siden) . Vet vi at dette fungerer nå? (Og hva er risiko hvis det ikke fungerer?)

Reply:
Relativt liten issue her da - vi sender i verste fall "dobbelt opp" med måledata hvis to blir leader - og litt lite måledata hvis `leader` av en eller annen grunn ikke finnes

Reply:
Det så ut til å fungere i familie-tilbake. Problemet uten er at den da summerer tellingen fra de to poddene, så alle tall i prometheus blir (riktig antall * antall pods).

Reply:
Ja, godt poeng. Med mindre "spørringen" i grafana klarer å skille på podene, men da vil den jo få en liste med svar, og ikke ett svar.

Reply:
Tror grafana blir lettere på denne måten. Er litt herk å få grupperingen til å fungere riktig allerede.

Reply:
Burde gå fint å bare gruppere? eks noe liknende som denne, så unngår man det med master/slave - føles litt unødvendig for å få ut metrics? 
![image](https://user-images.githubusercontent.com/937168/143288276-c429718b-220d-4cc7-bbef-c3ee7c09ee43.png)